src/components/thumbnail/collapsible-document-section.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { observer } from "mobx-react";
+import { NavTabSectionModelType } from "../../models/view/nav-tabs";
+import { DocumentModelType, getDocumentContext } from "../../models/document/document";
+import { DocumentContextReact } from "../document/document-context";
+import { TabPanelDocumentsSubSectionPanel } from "./tab-panel-documents-subsection-panel";
+import { DocumentCaption } from "./document-caption";
+import { IStores } from "../../models/stores/stores";
+import ArrowIcon from "../../assets/icons/arrow/arrow.svg";
+import NotSharedIcon from "../../assets/icons/share/not-share.svg";
+
+import "./tab-panel-documents-section.sass";
+import "./collapsible-document-section.scss";
+
+
+interface IProps {
+  userName: string;
+  classNameStr: string;
+  sectionDocs: DocumentModelType[];
+  section: NavTabSectionModelType;
+  stores: IStores;
+  tab: string;
+  scale: number;
+  selectedDocument?: string;
+  onSelectDocument?: (document: DocumentModelType) => void;
+}
+
+export const CollapsibleDocumentsSection: React.FC<IProps> = observer(({userName, classNameStr, sectionDocs, section,
+                                                                        stores, tab, scale, selectedDocument,
+                                                                        onSelectDocument}) => {
```

Comment:
You could also format this as:
```suggestion
export const CollapsibleDocumentsSection: React.FC<IProps> = observer(
  ({userName, classNameStr, sectionDocs, section, stores, tab, scale, selectedDocument, onSelectDocument}) => {
```